crates/miden-tx-batch-prover/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "miden-tx-batch-prover"
+version = "0.8.0"
+description = "Miden rollup batch executor and prover"
```

Comment:
nit: I would write this as: "Miden rollup transaction batch executor and prover"